### PR TITLE
Responsive avatars

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/SDS-243

#### Description
Avatars scale at the same rate we scale type and ([soon](https://github.com/meetup/meetup-web-components/pull/163)) icons

NOTE: This will break `AvatarRow` in `mup-web` - will need to update `height` to also use responsive media variables

### CODE REVIEW

File: assets/scss/components/_avatar.scss [hunk 1]
```diff
@@ -80,33 +80,33 @@ Class                     | Description
...
-	height: $media-m;
 	text-indent: 100%;
 	white-space: nowrap;
 	vertical-align: top;
-	width: $media-m;
 
-	.inverted & {
-		background: $C_textHintInverted;
+	@include responsiveVarContext--media() {
+		height: $dynamicMedia-m;
```
Comment (akdetrick) [inline assets/scss/components/_avatar.scss]: Now that i'm looking at usage, we should probably change this responsive var context mixin name to `responsiveVarContext--scalingMedia` and the vars to `scalingMedia-[size]`.

I can make the update to sasstools before we merge this PR.

Comment (akdetrick) [inline assets/scss/components/_avatar.scss]: sasstools PR here: https://github.com/meetup/swarm-sasstools/pull/13

Comment (mperrotti) [inline assets/scss/components/_avatar.scss]: Thanks. Reviewed and approved. Will update variable names

File: assets/scss/components/_avatar.scss [hunk 2]
```diff
@@ -80,33 +80,33 @@ Class                     | Description
...
 // Size variants
 //
-.avatar--small {
-	height: $media-s;
-	width: $media-s;
-}
-.avatar--big {
-	height: $media-xxl;
-	width: $media-xxl;
+@include responsiveVarContext--media() {
```
Comment (akdetrick) [inline assets/scss/components/_avatar.scss]: `responsiveVarContest--scalingMedia()`

**Comment by akdetrick** [comment]
Found a few more Sass mixin name errors testing locally. I'm going to fix those & merge, and follow up with a ticket to see if we can improve automated checks to fail on Sass compilation errors.

**Comment by akdetrick** [comment]
Pushed a fix - it was a bug in sasstools. Published sasstools `1.3.14` to fix the issue.

**Pull request merged**